governance/polls/Increase the GSM Pause Delay - March 25, 2024.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Increase the GSM Pause Delay - March 25, 2024
+summary: Signal your support or opposition to increasing the GSM Pause Delay
+discussion_link: https://forum.makerdao.com/t/gsm-pause-delay-increase-proposal/23929
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-03-25T16:00:00
+end_date: 2024-03-28T16:00:00
+---
+# Poll: Increase the GSM Pause Delay - March 25, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, March 25 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following GSM Pause Delay change:
```

Comment:
:yellow_circle: **Minor**: add link to MIP113 entry that defines the GSM Pause Delay

```suggestion
The community can vote in this poll to express support or opposition to the following [GSM Pause Delay](https://mips.makerdao.com/mips/details/MIP113#10-1-gsm-governance-security-module-pause-delay) change:
```